backend/capellacollab/sessions/operators/k8s.py

```diff
@@ -1094,6 +1095,47 @@ def reader():
             )
             raise
 
+    def delete_file(self, session_id: str, path: str):
+        normalized_path = os.path.normpath(path)
+        if normalized_path == "/workspace":
```

Comment:
I'd check that the path begins with `/workspace/` instead; it makes sure that we only delete files in `/workspace` and not in any other directory (possibly damaging a container).